projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -282,11 +288,12 @@ private function prepare_submenu_item( array $submenu_item, array $menu_item ) {
 		}
 
 		$item = array(
-			'parent' => sanitize_title_with_dashes( $menu_item[2] ),
-			'slug'   => sanitize_title_with_dashes( $submenu_item[2] ),
-			'title'  => $submenu_item[0],
-			'type'   => 'submenu-item',
-			'url'    => $this->prepare_menu_item_url( $submenu_item[2], $menu_item[2] ),
+			'parent'     => sanitize_title_with_dashes( $menu_item[2] ),
+			'slug'       => sanitize_title_with_dashes( $submenu_item[2] ),
+			'title'      => $submenu_item[0],
+			'type'       => 'submenu-item',
+			'url'        => $this->prepare_menu_item_url( $submenu_item[2], $menu_item[2] ),
+			'identifier' => $this->prepare_menu_item_identifier( $submenu_item[2] ),
```

Comment:
If would be great if `$submenu_item[2]` could be a named var to make it more self documenting.